rio/src/RIO/Partial.hs

```diff
@@ -0,0 +1,10 @@
+-- | Partial functions. Import as:
+--
+-- > import qualified RIO.Partial as P
```

Comment:
I don't think this requires a qualified import mention, since the names don't conflict. But _if_ we were to recommend one, I'd go for the full word `Partial`; we should make this as painful as possible 😛